optaplanner-examples/src/main/java/org/optaplanner/examples/conferencescheduling/persistence/ConferenceSchedulingGenerator.java

```diff
@@ -446,31 +454,75 @@ private void createTalkList(ConferenceSolution solution, int talkListSize) {
                     break;
                 }
             }
+            Set<String> requiredRoomTagSet = new LinkedHashSet<>();
+            Set<String> preferredRoomTagSet = new LinkedHashSet<>();
+            Set<String> prohibitedRoomTagSet = new LinkedHashSet<>();
+            Set<String> undesiredRoomTagSet = new LinkedHashSet<>();
+            setRoomTags(requiredRoomTagSet, preferredRoomTagSet, prohibitedRoomTagSet, undesiredRoomTagSet);
```

Comment:
vertical density:
Put the 5 lines above (which create and initialize the roomTagSets) right above the lines that do the setRequiredRoomTagSet().